sensor_msgs/Image image
int32 nbest
---
mask_rcnn/MaskArray masks
